src/Maintenance/Maintenance.Test/ScenarioTests/MaintenanceTests.ps1

```diff
@@ -12,6 +12,84 @@
 # limitations under the License.
 # ----------------------------------------------------------------------------------
 
+<#
+.SYNOPSIS
+Test New-AzMaintenanceConfiguration, Get-AzMaintenanceConfiguration, Remove-AzMaintenanceConfiguration
+#>
+function Test-GetAzMaintenanceConfiguration
+{
+    $resourceGroupName = Get-RandomResourceGroupName
+    $maintenanceConfigurationName = Get-RandomMaintenanceConfigurationName
+    $location = "eastus2euap"
+    $maintenanceScope = "Host"
+    $visibility = "Custom"
+    $startDateTime = "2024-08-05 12:30"
+    $timezone = "Pacific Standard Time"
+    $recurEvery = "Day"
+    $duration = "05:00"
+    $expirationDateTime = "9999-12-31 23:59"
+    
+    try
+    {
+        # Create Resource Group
+        New-AzResourceGroup -Name $resourceGroupName -Location $location
+        Write-Host "Created Resource Group $resourceGroupName in $location"
+
+        # Create Maintenance Configuration
+        $maintenanceConfiguration = New-AzMaintenanceConfiguration -ResourceGroupName $resourceGroupName -Name $maintenanceConfigurationName -MaintenanceScope $maintenanceScope -Location $location -Visibility $visibility -StartDateTime $startDateTime -Timezone $timezone -RecurEvery $recurEvery -Duration $duration -ExpirationDateTime $expirationDateTime
```

Comment:
The PR targets to fix issue that Update-AzMaintenanceConfiguration can't receive input from pipeline way if Get-AzMaintenanceConfiguration returns multiple maintenance configurations as list. Please include this scenario into tests.

Reply:
Thanks for pointing this out, I have added this scenario :)